challenge_3/javascript/Zirhc86/src/majority-element.js

```diff
@@ -0,0 +1,42 @@
+var array = [2,2,3,7,5,7,7,7,4,7,2,7,4,5,6,7,7,8,6,7,7,8,10,12,29,30,19,10,7,7,7,7,7,7,7,7,7];
+
+// Finds majority element using the Boyer-Moore majority vote algorithm.
+function findMajority(arr){
+	var count = 0, currentMajority;
+
+	for(var i =0; i<arr.length; i++){
+		if(count ==0){
+			currentMajority = arr[i];
+			count=1;			
+		}
+
+		if(arr[i] == currentMajority){
+			count++;
+		}
+		else{
+			count--;
+		}
+	}
+
+	count=0;
+
+	for(var i =0; i<arr.length; i++){
```

Comment:
Could probably use a space after the `=`, or remove the space before, your choice.